src/screens/CreatePost/index.js

```diff
@@ -752,7 +751,6 @@ const CreatePost = () => {
             topicChats={listTopicChat}
             allTaggedUser={allTaggingUser}
             setAllTaggedUser={setAllTaggingUser}
-            isAnonimity={typeUser}
           />
           {typeUser && (
             <Animated.View style={[{opacity: animatedReminder}, styles.reminderContainer]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `isAnonimity` has been removed. If this prop was used in the child component, ensure it's no longer needed or is being provided through a different mechanism.
```diff
- isAnonimity={typeUser}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->